lib/data/monorepo.json

```diff
@@ -382,7 +382,7 @@
     "jna": "https://github.com/java-native-access/jna",
     "json-smart-v2": "https://github.com/netplex/json-smart-v2",
     "jsplumb": "https://github.com/jsplumb/jsplumb",
-    "junit5": "https://github.com/junit-team/junit5",
+    "junit-framework": "https://github.com/junit-team/junit-framework",
```

Comment:
This is technically a breaking change. Users that reference the `monorepo:junit5` preset will run into issues.

Reply:
Should we add it as a new repo instead?

Reply:
we need a migration. we've already some preset renamings. please check them.